pkg/kubelet/volumemanager/cache/actual_state_of_world.go

```diff
@@ -265,6 +264,14 @@ type attachedVolume struct {
 	// deviceMountPath contains the path on the node where the device should
 	// be mounted after it is attached.
 	deviceMountPath string
+
+	// claimStatusSize indicates actual size of the persistent volume claim.
+	// This is mainly calculated from pvc.Status.Capacity
```

Comment:
Can you add some more comments here explaining how this is populated/updated, how it addresses the RWX case, and the known races?

Reply:
done.